src/activity_actor.cpp

```diff
@@ -5730,7 +5730,7 @@ void outfit_swap_actor::finish( player_activity &act, Character &who )
     // Taken-off items are put in this temporary list, then naturally deleted from the world when the function returns.
     std::list<item> it_list;
     for( item_location &worn_item : who.get_visible_worn_items() ) {
-        item outfit_component( *worn_item );
+        const item &outfit_component( *worn_item );
```

Comment:
Is this safe? does taking off the worn item not invalidate the item location and then invalidate the outfit component?